src/bot/plugins.py

```diff
@@ -1,7 +1,16 @@
+import re
+
+from dependency_injector.wiring import Container, Provide, inject
 from mmpy_bot import Message, Plugin, listen_to
 
+from src.bot.services.registration import RegistrationService
+
 
-class MyPlugin(Plugin):
-    @listen_to("Привет")
-    async def wake_up(self, message: Message):
-        self.driver.reply_to(message, "Привет, я помогу организовать тебе random coffee")
+@inject
+class Registration(Plugin):
+    @listen_to("Register", re.IGNORECASE)
+    async def Register(
+        self, message: Message, registration: RegistrationService = Provide[Container.registration_service]
+    ):
+        await registration.register(**self.driver.get_user_info(message.user_id))
```

Comment:
Очень плахая практика передавать переменные таким образом. Лучше распарсь их (можно использовать для этого pydantic) чтобы поймать ошибку если нет нужных данных или они не валидны, а затем передать в функцию только те что требуются:

```suggestion
        await registration.register(
            first_name=data.first_name,
            last_name=data.last_name,
            ...
        )
```